server/server_test.go

```diff
@@ -308,45 +312,52 @@ func TestGinPostCreateInteraction(t *testing.T) {
 		assert.Equal(t, `Updated player interaction`, msg)
 	}
 }
+
+func createTestGame(t *testing.T, cs *cribbageServer, pIDs []model.PlayerID) model.Game {
+	ctx := context.Background()
+	db, err := cs.dbFactory.New(ctx)
+	require.NoError(t, err)
+	defer db.Close()
+	g, err := createGame(ctx, db, pIDs)
+	require.NoError(t, err)
+	return g
+}
```

Comment:
🌵 since thisi is intended to only be used within the `func TestGinGetGame`, how about we make it a var that is an anonymous func within that scope instead? (Typically, I don't like passing funcs around, but to scope behavior to a specific test seems like an appropriate reason)

```go
func TestGinGetGame(t *testing.T) {
  createTestGame := func(cs *cribbageServer, pIDs []model.PlayerID) model.Game {
     ...
  }
  testCases := ...
```